pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -193,10 +193,6 @@ func (n *NodeLabeller) run() error {
 
 	node := originalNode.DeepCopy()
 
-	if skipNode(node) {
```

Comment:
not related to the PR: It would be great if we can move this skip before the `DeepCopy`.